### ISSUE TITLE
Total Estimation overview

### ISSUE THREAD

**Issue by zsculpt** [SoftFever/OrcaSlicer#6233]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

It drives me insane that I can't see the total time estimation when I'm looking at any other views besides "Line Type", in particular the "Speed" and "Layer Time" views. 

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

Please make the total estimation sub-view available to all preview modes (Speed, Layer Time, etc.) instead of only being visible under "Line Type"

**Comment by DVSVIDEO** [comment]
Right! That annoys me too...

**Comment by kylek29** [comment]
Agreed. Ideally it'd have a collapsible panel of details that is separate from the non-view specific stats.

It'd be nice to have it do breakdown by plate and objects on that plate.

**Comment by t-moennich** [comment]
bump

**Comment by fritzw** [comment]
Bump

**Comment by davidmessem** [comment]
+1, would really appreciate this
